docs/api/public_php_api.md

```diff
@@ -702,6 +702,10 @@ Query object contains two properties you can set criteria on, `filter` and `quer
 
 #### Performing a simple full text search
 
+!!! tip "Checking feature support per search engine"
+
+    To find out if a given search engine supports advanced full text capabilities, use the [`supports($capabilityFlag)`](https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/SPI/Search/Capable.php#L25) method.
```

Comment:
link is wrong, it should point to API not SPI for the intended audience here.

Reply:
https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/API/Repository/SearchService.php#L187-L197

SIDE: Linking to master is a bit risky, better to swap master for 6.12 once we have a branch for it.